backend/coins/btc/handlers/handlers.go

```diff
@@ -195,6 +200,76 @@ func (handlers *Handlers) getAccountTransactions(_ *http.Request) (interface{},
 	return result, nil
 }
 
+func exportFilePath(accountName string) string {
+	path := os.Getenv("HOME")
+	switch runtime.GOOS {
+	case "darwin":
+		path = filepath.Join(path, "Downloads")
+	case "windows":
+		path = filepath.Join(path, "Downloads")
+	case "linux":
+		path = filepath.Join(path, "Downloads")
+	}
+	path = filepath.Join(path, time.Now().Format("2006-01-02 at 15-04-05 ")+accountName+" Export.csv")
+	return path
+}
+
+func (handlers *Handlers) postExportTransactions(_ *http.Request) (interface{}, error) {
+	path := exportFilePath(handlers.account.Name())
+	handlers.log.Infof("Export transactions to %s.", path)
+
+	file, err := os.Create(path)
+	if err != nil {
+		return nil, errp.WithStack(err)
+	}
+	defer func() {
+		err := file.Close()
+		if err != nil {
+			handlers.log.WithError(err).Error("Could not close the exported transactions file.")
+		}
+	}()
+
+	writer := csv.NewWriter(file)
+	defer writer.Flush()
+
+	err = writer.Write([]string{
+		"Time",
+		"Type",
+		"Amount",
+		"Fee",
+		"Address",
+		"Transaction ID",
+	})
+	if err != nil {
+		return nil, errp.WithStack(err)
+	}
+
+	for _, transaction := range handlers.account.Transactions() {
+		transactionType := map[coin.TxType]string{
```

Comment:
just `string(transaction.Type())` might work

Reply:
I just "blindly" copied your code a few lines higher: https://github.com/digitalbitbox/bitbox-wallet-app/blob/70d12a3f0eb671a6eabadb96ec8e920ddce1599d/backend/coins/btc/handlers/handlers.go#L171-L175

I think I leave it for now because I changed the label to past tense.